src/main.rs

```diff
@@ -203,6 +203,19 @@ fn index() -> Node {
                     black-logo-square.svg
                 </a>
             </li>
+
+            <h3 class="text-l"><b>Automatic (respects users' Dark Mode / Light Mode preferences)</b></h3>
+            <ul class="list-disc">
+            <li>
+                <a href="automatic-logo.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
+                    automatic-logo.svg
+                </a>
+            </li>
+            <li>
+                <a href="automatic-logo-square.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
+                    automatic-logo-square.svg
+                </a>
+            </li>
```

Comment:
This is how this new section renders:

![image](https://github.com/user-attachments/assets/e79fe845-319d-4b37-a277-d657df94f31e)

I'm not sold on the text in parenthesis, I've be ok just saying "White, Black, Automatic" but I don't know if that's clear enough?

Reply:
```suggestion

            <h3 class="text-l"><b>Automatic</b></h3>
            <ul class="list-disc">
            <li>
                <a href="automatic-logo.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
                    automatic-logo.svg
                </a>
            </li>
            <li>
                <a href="automatic-logo-square.svg" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
                    automatic-logo-square.svg
                </a>
            </li>
```